src/main/java/bio/terra/service/snapshotbuilder/query/SubstituteVariable.java

```diff
@@ -0,0 +1,14 @@
+package bio.terra.service.snapshotbuilder.query;
+
+public class SubstituteVariable implements SelectExpression {
+  private final String searchText;
+
+  public SubstituteVariable(String searchText) {
+    this.searchText = searchText;
+  }
+
+  @Override
+  public String renderSQL(SqlRenderContext context) {
+    return context.getPlatform().choose("@" + searchText, ":" + searchText);
```

Comment:
Can you add a comment that links to the APIs where these conventions are documented? URLs would be fine too.

maybe https://docs.spring.io/spring-framework/reference/data-access/jdbc/core.html#jdbc-NamedParameterJdbcTemplate

and https://cloud.google.com/java/docs/reference/google-cloud-bigquery/latest/com.google.cloud.bigquery.QueryJobConfiguration.Builder#com_google_cloud_bigquery_QueryJobConfiguration_Builder_setNamedParameters_java_util_Map_java_lang_String_com_google_cloud_bigquery_QueryParameterValue__

Reply:
thanks for providing the links!